addons/website/views/snippets/snippets.xml

```diff
@@ -661,7 +664,7 @@
     <div data-js="layout_column"
         data-selector="section, section.s_carousel_wrapper .carousel-item"
         data-target="> *:has(> .row), > .s_allow_columns"
-        data-exclude=".s_masonry_block, .s_features_grid, .s_media_list, .s_table_of_content, .s_process_steps, .s_image_gallery, .s_timeline">
+        data-exclude=".s_masonry_block, .s_features_grid, .s_media_list, .s_table_of_content, .s_process_steps, .s_image_gallery, .s_timeline, .s_images_constellation">
```

Comment:
Since you're removing the ability to switch to col, it is still important to be able to add new images/text/button. so a few lines above, add the snippet in the data-selector.

```xml
 <!-- Grid only -->
<div data-js="layout_column"
    data-selector="section.s_masonry_block, section.s_images_constellation"
    data-target="> *:has(> .row)">
	<t t-call="website.grid_layout_options"/>
</div>
```